index.js

```diff
@@ -43,7 +43,16 @@ var excludedKeys = {
 	$webkitStorageInfo: true,
 	$window: true
 };
+var isChromeExtension = (function () {
+	/* global chrome */
+	return (typeof chrome !== 'undefined') &&
+		(typeof chrome.runtime !== 'undefined') &&
+		(typeof chrome.runtime.id !== 'undefined');
+}());
 var hasAutomationEqualityBug = (function () {
+	if (isChromeExtension) {
+		return false;
```

Comment:
Also this issue

https://chromium.googlesource.com/chromium/src/+/master/extensions/renderer/resources/platform_app.js#43